src/main/cpp/Android.mk

```diff
@@ -66,6 +66,9 @@ processors/pitchshifter.cpp \
 processors/reverb.cpp \
 processors/tremolo.cpp \
 processors/waveshaper.cpp \
+processors/comb.cpp \
+processors/allpass.cpp \
+processors/revmodel.cpp \
```

Comment:
As _comb_ and _allpass_ are here very specific to _revmodel_ might as well inline their class definitions inside the _revmodel_ header and implementation files.